backend/src/test/java/com/woowacourse/gongcheck/core/application/JobServiceTest.java

```diff
@@ -65,272 +70,518 @@ class JobServiceTest {
-    @Test
-    void 존재하지_않는_Space의_Job_목록을_조회할_경우_예외를_던진다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
+        @Nested
+        class 존재하지_않는_Host로_조회할_경우 {
 
-        assertThatThrownBy(() -> jobService.findJobs(host.getId(), 0L))
-                .isInstanceOf(NotFoundException.class)
-                .hasMessage("존재하지 않는 공간입니다.");
-    }
+            @Test
+            void 예외를_발생시킨다() {
+                assertThatThrownBy(() -> jobService.findJobs(0L, 1L))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 호스트입니다.");
+            }
+        }
 
-    @Test
-    void 다른_Host의_Space의_Job_목록을_조회할_경우_예외를_던진다() {
-        Host host1 = hostRepository.save(Host_생성("1234", 1234L));
-        Host host2 = hostRepository.save(Host_생성("1234", 2345L));
-        Space space = spaceRepository.save(Space_생성(host2, "잠실"));
+        @Nested
+        class 존재하지_않는_Space의_Job_목록을_조회할_경우 {
 
-        assertThatThrownBy(() -> jobService.findJobs(host1.getId(), space.getId()))
-                .isInstanceOf(NotFoundException.class)
-                .hasMessage("존재하지 않는 공간입니다.");
-    }
+            private Long hostId;
 
-    @Test
-    void Job과_Section들과_Task들을_한_번에_생성한다() {
-        Host host = hostRepository.save(Host_생성("1234", 1234L));
-        Space space = spaceRepository.save(Space_생성(host, "잠실"));
-        List<TaskCreateRequest> tasks = List.of(new TaskCreateRequest("책상 닦기"), new TaskCreateRequest("칠판 닦기"));
-        List<SectionCreateRequest> sections = List.of(new SectionCreateRequest("대강의실", tasks));
-        JobCreateRequest jobCreateRequest = new JobCreateRequest("청소", sections);
+            @BeforeEach
+            void setUp() {
+                hostId = hostRepository.save(Host_생성("1234", 1234L))
+                        .getId();
+            }
 
-        Long savedJobId = jobService.createJob(host.getId(), space.getId(), jobCreateRequest);
+            @Test
+            void 예외를_발생시킨다() {
+                assertThatThrownBy(() -> jobService.findJobs(hostId, 0L))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 공간입니다.");
+            }
+        }
 
-        assertThat(savedJobId).isNotNull();
+        @Nested
+        class 다른_Host의_Space의_Job_목록을_조회할_경우 {
+
+            private Host otherHost;
+            private Space space;
+
+            @BeforeEach
+            void setUp() {
+                Host host = hostRepository.save(Host_생성("1234", 1234L));
+                otherHost = hostRepository.save(Host_생성("1234", 2345L));
+                space = spaceRepository.save(Space_생성(host, "잠실"));
+            }
+
+            @Test
+            void 예외를_발생시킨다() {
+                assertThatThrownBy(() -> jobService.findJobs(otherHost.getId(), space.getId()))
+                        .isInstanceOf(NotFoundException.class)
+                        .hasMessage("존재하지 않는 공간입니다.");
+            }
+        }
```

Comment:
```suggestion
        class 다른_Host의_Space의_Job_목록을_조회할_경우 {

            private Host host;
            private Space space;

            @BeforeEach
            void setUp() {
                host = hostRepository.save(Host_생성("1234", 1234L));
                Host otherHost = hostRepository.save(Host_생성("1234", 2345L));
                space = spaceRepository.save(Space_생성(otherHost, "잠실"));
            }

            @Test
            void 예외를_발생시킨다() {
                assertThatThrownBy(() -> jobService.findJobs(host.getId(), space.getId()))
                        .isInstanceOf(NotFoundException.class)
                        .hasMessage("존재하지 않는 공간입니다.");
            }
        }
```
host가 테스트하는 주체가되고 다른 호스트의 space를 진행한다라는 행위이기 때문에 이렇게 변경하는게 어떨까요?